Author: Guillem Jover <guillem@debian.org>
Description: Fix cross-compilation on musl-based systems.
 The gettext implementation in musl does not define some of the internal
 symbols that AM_GNU_GETTEXT used to check for. This got fixed in gettext
 0.19.8, but the AM_GNU_GETTEXT_VERSION macro requires a specific version
 even if the system contains a higher one. We switch to use the new
 AM_GNU_GETTEXT_REQUIRE_VERSION macro which requires at least a specific
 version, and bump the AM_GNU_GETTEXT_VERSION to the first one that had
 support for the new macro.
Origin: vendor
Forwarded: yes
Applied-Upstream: 2.3.1+, commit:97a8a4fbda04af2c6f9a1cfc95eb780ae9fe36d8

---
 configure.ac |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

--- a/configure.ac
+++ b/configure.ac
@@ -34,7 +34,10 @@ AC_SYS_LARGEFILE
 AM_PROG_AR
 LT_INIT
 
-AM_GNU_GETTEXT_VERSION([0.18.2])
+# Minimal version supporting AM_GNU_GETTEXT_REQUIRE_VERSION.
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
+# Require at least the following version, but use the latest available one.
+AM_GNU_GETTEXT_VERSION([0.19.6])
 AM_GNU_GETTEXT([external])
 
 AC_ARG_ENABLE([debug],
